src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/paragraph/helpers.js

```diff
@@ -27,6 +27,10 @@ export const processBlock = _block => {
   const block = _block;
 
   if (path(['text'], block)) {
+    // catch plain_text ARES response for legacy MAPs with <link> elements
+    if (block.text.includes('</link>')) {
+      block.markupType = 'candy_xml';
```

Comment:
Please can you test this with a link that contains chevrons or ampersands?

Reply:
Quotations are encoded in the desired format - `&quot;` so we're optimistic that the other characters will be too. Would need to test real TC2 responses with ampersands/chevrons to see what comes back in the response but I think we're confident enough for now. Examples of the `&quot;` encoding for legacy assets below. 

`/simorgh/data/afrique/legacyAssets/nos_emissions/2014/11/141111_porno_vengeur_ouganda.json`

`/simorgh/data/persian/legacyAssets/arts/2011/08/110802_doublage_sherlock_promo.json`

`/simorgh/data/portuguese/legacyAssets/noticias/2016/03/160307_andes_vivos_tg.json`